mypy/fastparse.py

```diff
@@ -213,10 +214,64 @@ def parse(
         assert options.python_version[0] >= 3
         feature_version = options.python_version[1]
     try:
+        ignore_comment_errors = options and options.ignore_comment_errors
```

Comment:
```suggestion
        ignore_comment_errors = options.ignore_comment_errors
```
Or do we need to check that `options` are truthy here?

Reply:
I think we do, b/c option: `Options | None = None`.

Reply:
Don't we reinitialize it above?
https://github.com/python/mypy/blob/9fbcfaeea9b678cecaf54c73b38a4f084d7e0308/mypy/fastparse.py#L202-L203
In the same function a few lines above we're using `options` without any `None` testing.

Reply:
Yes, you're right. Same thing with `errors`. My IDE was complaining about it, that's why I put in the check. But `mypy` doesn't, so I'll remove those checks.